api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/DefaultDeleteGenerator.java

```diff
@@ -28,7 +28,9 @@
 	private Integer dataHash = 0;
 	private DataPropertyStatement dps = null;
 	private String dataLiteral = null;
-	private String template = "confirmDeletePropertyForm.ftl";
+  private String propertyTemplate = "confirmDeletePropertyForm.ftl";
+  private String individualTemplate = "confirmDeleteIndividualForm.ftl";
```

Comment:
Can you please correct lay outing of those two lines?

Reply:
sure